WcaOnRails/config/application.rb

```diff
@@ -1,5 +1,6 @@
 require File.expand_path('../boot', __FILE__)
-require File.expand_path('../../lib/fix_accept_header', __FILE__)
+require_relative '../lib/middlewares/fix_accept_header'
```

Comment:
Just curious, why go with `require_relative` here, but leave the first line alone?